.Rbuildignore

```diff
@@ -2,3 +2,5 @@ cran-comments.md
 ^revdep$
 .travis.yml
 ^CRAN-RELEASE$
+^.*\.Rproj$
```

Comment:
Great. Can you also remove the file `socialmixr.Rproj`, please.

Reply:
Removed